tests/TaskFieldRenderer.test.ts

```diff
@@ -35,16 +34,15 @@ describe('Field Layouts Container tests', () => {
 
         fieldRenderer.addDataAttribute(span, task, 'priority');
 
-        expect(Object.keys(span.dataset).length).toEqual(1);
-        expect(span.dataset['taskPriority']).toEqual('medium');
+        expect(span).toHaveDataAttributes('taskPriority: medium');
     });
     it('should not add any data attributes for a missing component', () => {
         const task = new TaskBuilder().build();
         const span = document.createElement('span');
 
         fieldRenderer.addDataAttribute(span, task, 'recurrenceRule');
 
-        expect(Object.keys(span.dataset).length).toEqual(0);
+        expect(span).toHaveDataAttributes('');
```

Comment:
How does this test distinguish between these two situations:

- No value in `dataset`
- A single value in `dataset` which is an empty string.

Reply:
Good point =)

No value in dataset -> the expected data attributes will be `''`
A single value in dataset which is an empty string -> the expected data attributes will be `'singleValue: '` (note the absence of the data attribute value after the column)

Reply:
Oh thanks.